modules/system/version.nix

```diff
@@ -59,11 +59,17 @@ in
       description = "Label to be used in the names of generated outputs.";
     };
 
+    system.darwinRelease = mkOption {
+      type = types.str;
+      default = (lib.importJSON ../../version.json).release;
+      description = "The nix-darwin release (e.g. `24.11`).";
+    };
```

Comment:
This should be read-only, right?

Reply:
Yes, `system.nixos.release`, which is read only (so I’ve fixed that). The inconsistent naming here is unfortunate, but I thought it better to stick with the existing convention throughout the file and address all of them at once at some later date. (Though I do admit to feeling a little bad about introducing a new option with a name I don’t love.)

Reply:
It would be good to just add a comment to document the equivalent option

Reply:
This is just another entry in the various version options (`system.darwinLabel`, `system.darwinVersion`, `system.darwinVersionSuffix`, etc.) that all mirror various NixOS options. Do you want comments on all of those too? Maybe it should be in a different PR?

Reply:
Sure a different PR is fine